app/controllers/api/v1/packages_controller.rb

```diff
@@ -75,6 +75,26 @@ def destroy
       render json: {}
     end
 
+    api :POST, "/v1/packages/print_barcode", "Print barcode"
+    def print_barcode
+      begin
+        package = Package.find params[:package_id]
+      rescue ActiveRecord::RecordNotFound
+        return render json: {errors:"Package not found with supplied package_id"}, status: 400
+      end
```

Comment:
Any specific reason to use begin-rescue block here? It can be handled here https://github.com/crossroads/api.goodcity/blob/master/app/controllers/api/v1/api_controller.rb#L6 right?
cc: @mcm-ham @steveyken